packages/redux/src/recentlyViewed/actions/factories/removeRecentlyViewedProductFactory.ts

```diff
@@ -0,0 +1,45 @@
+import * as actionTypes from '../../actionTypes';
+import type { DeleteRecentlyViewedProducts } from '@farfetch/blackout-client/src/recentlyViewed/types';
+import type { Dispatch } from 'redux';
+import type { RemoveRecentlyViewedProductAction } from '../../types';
+import type { RemoveRecentlyViewedProductsFactory } from './types';
+
+/**
+ * Method to create a thunk factory configured with the specified client for deleting a recently viewed product, both locally and on the server.
+ *
+ * @param deleteRecentlyViewedProduct - Delete recently viewed client.
+ *
+ * @returns Thunk factory.
+ */
+
+const removeRecentlyViewedProductFactory: RemoveRecentlyViewedProductsFactory<DeleteRecentlyViewedProducts> =
+
+    deleteRecentlyViewedProduct =>
+    (productId, config) =>
+    async (
+      dispatch: Dispatch<RemoveRecentlyViewedProductAction>,
+    ): Promise<void> => {
```

Comment:
This void type does not match to the unknown type defined for the return value of the deleteRecentlyViewedProduct argument.

Reply:
Can you check again. I update Type from Promise<unknown> to Promise<void>